tests/integration/commands/generateToken.spec.ts

```diff
@@ -1,10 +1,11 @@
 import { writeFile } from 'fs/promises';
-import { EOL } from 'os';
 import path from 'path';
 import { executeCli } from '../../helpers/execute';
 import { PRIVATE_KEY } from '../../keys';
 import { CLIENT_NAME, FILE_STORAGE_DIR } from '../../testConstants';
 
+const EOL = '\n';
```

Comment:
why not EOL based on `os`?

Reply:
better export it to a const instead of assiging the variable in each file

Reply:
fixed